mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/ui/social/SocialScreenTest.kt

```diff
@@ -23,14 +24,18 @@ class SocialScreenTest {
   @get:Rule val rule = createComposeRule()
 
   private class SnapshotSocialScreenState : SocialScreenState {
+    override var searchResult: List<Person> = emptyList()
     override var mode: SocialScreenState.Mode by mutableStateOf(Following)
-    override var players: List<Person> =
+    override var following: List<Person> =
         listOf(
             createPerson(Color.Default, "Toto", ":)"),
             createPerson(Color.Default, "John", ":3"),
             createPerson(Color.Default, "Travis", ";)"),
             createPerson(Color.Default, "Cirrus", "TwT"))
     override var input: String by mutableStateOf("")
+    override var searchFieldInteraction: MutableInteractionSource
+      get() = MutableInteractionSource()
+      set(value) {}
```

Comment:
Nitpick : I think the `MutableInteractionSource` should be a property `val`.

```suggestion
    override val searchFieldInteraction = MutableInteractionSource()
```